store/sqlstore/team_store.go

```diff
@@ -1063,6 +1065,7 @@ func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, teamMembe
 	return dbMembers.ToModel(), nil
 }
 
+// GetTotalMemberCount returns the number of all members in a team for the teamId passed as a parameter.
```

Comment:
I would expect here to see something about the `restrictions` parameter. Something like "The restrictions parameters allows you to define the set of channels and teams that are visible to the caller of the query, and applies that restrictions to the results."

I would expect something similar in the other places where `restrictions` is passed as parameter.